packages/@react-types/progress/src/index.d.ts

```diff
@@ -14,37 +14,74 @@ import {DOMProps, LabelPosition, StyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
 interface ProgressBaseProps {
+  /**
+   * The current value (controlled).
+   * @default "0"
+   */
   value?: number,
+  /**
+   * The smallest value allowed for the input.
+   * @default "0"
+   */
   minValue?: number,
+  /**
+   * The largest value allowed for the input.
+   * @default "100"
+   */
   maxValue?: number
 }
 
 export interface ProgressBarBaseProps extends ProgressBaseProps {
+  /** The content to display as the label. */
   label?: ReactNode,
+  /** Whether the value's label is displayed */
   showValueLabel?: boolean, // true by default if label, false by default if not
+  /** The display format of the value label. */
```

Comment:
Might be overkill since we mention it in the docs, but might be nice to have a inline link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat

Reply:
We decided this should be a link on the type as part of https://jira.corp.adobe.com/browse/RSP-1575